content/en/integrations/guide/aws-cloudwatch-metric-streams-with-kinesis-data-firehose.md

```diff
@@ -131,6 +131,15 @@ Once you see the Metric Stream resource has been successfully created, wait five
 {{% /tab %}}
 {{< /tabs >}}
 
+### Disabling Metric Streaming
+
+In order to disable metric streaming, you must delete the AWS metric stream resources directly on the <a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Metric-Streams.html" target="_blank">AWS Console</a>. In order to prevent loss of data, the following resources will need to be deleted for streaming to be correctly disabled:
```

Comment:
Revise to:
In order to disable metric streaming completely for a given AWS account and region, you must delete the AWS Metric Steam and its related resources. In order to prevent loss of metrics in Datadog, it's important to follow these deletion steps carefully: